pkg/import/import_resources.go

```diff
@@ -74,11 +74,14 @@ func Import(rd, definitionsFile io.Reader) ([]string, []string, error) {
 		// Get resource address
 		resAddr := fmt.Sprintf("'%s'", resource.Address)
 		// Proceed only if type is declared in resources definitions
+		// QUESTIONS:
+		// - why do we print a warning and break, instead of returning the warning msg as an error?
 		if _, ok := configs[resource.Type]; !ok {
 			msg := fmt.Sprintf("Warning: resource %s is not defined. Check %s documentation\n", resource.Type, resource.ProviderName)
 			fmt.Printf("\033[1;33m%s\033[0m", msg)
 			break
 		}
```

Comment:
@Esysc 
for my understanding, why do we print a warning and break from the loop, instead of returning the warning msg directly as an error, for consistency with the rest of the code?
This is the only place that prints something to stdout.

Reply:
This piece of code tries to match the resource in terraform plan with the definition file. If nothing has been found it prints out a warning and jump to the next resource.
When I implemented this block,  I reasoned following this logic:
- Loop starts analyzing resource A
- look for resource A in definition file
- resource A has not been found, warns the user and stops code execution for resource A
- Loop continue, resource B
- .....

So my thought; `terravalet` should continue for all the others resources it could know.